game/commander/objectivefinder.py

```diff
@@ -228,13 +228,16 @@ def farthest_friendly_control_point(self) -> ControlPoint:
             raise RuntimeError("Found no friendly control points. You probably lost.")
         return farthest
 
-    def closest_friendly_control_point(self) -> ControlPoint:
+    def closest_friendly_control_point(self) -> ControlPoint | None:
         """Finds the friendly control point that is closest to any threats."""
         threat_zones = self.game.threat_zone_for(not self.is_player)
 
         closest = None
         min_distance = meters(math.inf)
         for cp in self.friendly_control_points():
+            # If statement for https://github.com/dcs-liberation/dcs_liberation/issues/2693
+            if cp.is_fleet:
+                return closest
```

Comment:
I'm not entirely sure I understand the reasons behind this one 😅

Reply:
If we separate this into its own flight task these changes won't be needed which is what I think I am going to do